apps/client/src/features/editors/welcome/Welcome.tsx

```diff
@@ -84,6 +85,17 @@ export default function Welcome(props: WelcomeProps) {
             </div>
           </div>
           <div className={style.buttonRow}>
+            <Button
+              size='sm'
+              variant='ontime-subtle'
+              onClick={() => {
+                postShowWelcomeDialog(false);
+                handleClose();
+              }}
+            >
+              <ImportProjectButton onFinish={handleClose} />
+              Do not show welcome message again
+            </Button>
```

Comment:
suggestion: buttons inside buttons is usually a bad idea. I can help with this later on

we should also clarify the feature to when is this shown, I would suggest `Show welcome message on next start` or `Do not show this modal again`

Reply:
> buttons inside buttons is usually a bad idea. I can help with this later on

that was unintentional